bundle/manifests/node-healthcheck-operator.clusterserviceversion.yaml

```diff
@@ -223,15 +223,26 @@ spec:
     Remediation” API to instantiate the configured remediation template which
     specifies the mechanism/controller to be used.
 
-    ### Failure handling: Self Node Remediation
-    By default NHC depends on the “Self Node Remediation” (SNR) operator, which
-    is installed automatically.
+    ### Failure handling: External Remediators
+    NHC is part of the Medik8s team that has different remediators for handling node failure,
```

Comment:
- NHC is part of our team? :thinking:
- NHC is already explained in the chapter above :shrug:

Reply:
NHC is in our spirit. Bad wording indeed ;)